src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Machine {
+
+    public void doWork() {
+        System.out.println("Machine started working");
+    }
+
+    public void stopWork() {
+        System.out.println("Machine stopped working");
+    }

```

Comment:
methods are not `abstract`
![image](https://github.com/mate-academy/jv-oop/assets/73041091/f0f77940-23f3-4d09-a0fb-5ab070156b32)